index.js

```diff
@@ -41,7 +47,18 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+function getHighestMetascore(movies) {
+  if (!movies.length){
```

Comment:
Line 51 works for an empty array we  should look to also consider edge cases like someone passing in an empty object or undefined as well.